packages/x-charts/src/ChartsLabel/ChartsLabelGradient.tsx

```diff
@@ -118,11 +127,12 @@ const ChartsLabelGradient = consumeThemeProps(
   function ChartsLabelGradient(props: ChartsLabelGradientProps, ref: React.Ref<HTMLDivElement>) {
     const { gradientId, direction, classes, className, rotate, reverse, thickness, ...other } =
       props;
+    const isRtl = useRtl();
```

Comment:
Added on the `ChartsLabelGrandient` rather than the `ContinuousLegend` as doing so this change is used anywhere we use the gradient, if we use it somewhere else some day 🫠